locations/spiders/benchmark_hospitality.py

```diff
@@ -47,13 +53,16 @@ def parse_hotel(self, response):
             address = p[0]
             phone = ""
 
-        properties.update(
-            {
-                "ref": re.search(r".com/+(?:meeting/+)?(.+?)//?", response.url + "/").group(1),
-                "addr_full": address.strip() if "Reservations" not in address else None,
-                "phone": phone.strip(),
-                "website": response.url,
-            }
-        )
+        item["ref"] = re.search(r".com/+(?:meeting/+)?(.+?)//?", response.url + "/").group(1)
+        item["addr_full"] = address.strip() if "Reservations" not in address else None
+        item["phone"] = (phone.strip(),)
```

Comment:
This means the phone will show up as a list in the output JSON. We usually only include a single phone number.

```suggestion
        item["phone"] = phone.strip()
```